packages/react-native-layout/src/components/spacing/Spacing.props.ts

```diff
@@ -0,0 +1,3 @@
+export type SpacingProps =
```

Comment:
because of the sorting this shows in the PR as first props, then test then code. which makes it harder to stay in context